### PR TITLE
List and search second factors

### PR DESCRIPTION
**Pull request by rjkip**
With 2-item pages:

image

### CODE REVIEW

**Comment by rjkip** [comment]
Sort link changed to include header title with 97dc2bb:

image

**Comment by rjkip** [comment]
:bell: @DRvanR

File: app/AppKernel.php [hunk 1]
```diff
@@ -40,6 +40,7 @@ public function registerBundles()
             new JMS\TranslationBundle\JMSTranslationBundle(),
             new JMS\DiExtraBundle\JMSDiExtraBundle($this),
             new JMS\AopBundle\JMSAopBundle(),
+            new Knp\Bundle\PaginatorBundle\KnpPaginatorBundle(),
```
Comment (DRvanR) [inline app/AppKernel.php]: We're using Pagerfanta in the middleware, why opt for a different lib here?

File: src/Twig/Extension/XDebugExtension.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupRa\RaBundle\Twig\Extension;
+
+use Twig_Extension as Extension;
+use Twig_SimpleFunction as SimpleFunction;
+
+final class XDebugExtension extends Extension
```
Comment (DRvanR) [inline src/Twig/Extension/XDebugExtension.php]: Wait wut?

**Comment by DRvanR** [comment]
:+1:

**Pull request merged**